web/libs/editor/src/tags/visual/Filter.jsx

```diff
@@ -59,6 +59,11 @@ const Model = types
     get toTag() {
       return self.annotation.names.get(self.toname);
     },
+
+    // Indicates that it could exist without information about objects, taskData and regions
+    get isIndependent() {
+      return self.toTag ? self.toTag.isIndependent : false;
```

Comment:
same as this no?
```suggestion
      return self.toTag && self.toTag.isIndependent;
```

Reply:
@yyassi-heartex your version returns mixed values instead of plain boolean, so I'd prefer the original version

Reply:
ahh i see - good point

Reply:
@hlomzik From the other side, for some tags it can be `undefined` anyhow.

Reply:
since its really just checking if toTag is defined maybe we can do something like - feels like thats the intent - is that spot on?
```suggestion
      return self.toTag?.isIndependent ?? false;
```